tests/unit/store/playback.spec.js

```diff
@@ -593,83 +594,75 @@ describe("store/playback", () => {
 
       expect(store.state.flask.status_uuid).toEqual(expected_status_state);
     });
-    describe("When start_calibration is called", () => {
-      beforeEach(() => {
-        jest.restoreAllMocks();
-      });
-      test("Then playback state mutates to calibrating and starts status_pinging in Flask, then playback state mutates to calibrated", async () => {
-        let baseurl = "http://localhost:4567";
-        let api = "start_calibration";
 
-        mocked_axios
-          .onGet(system_status_when_calibrating_regexp)
-          .reply(200, { ui_status_code: STATUS.MESSAGE.STOPPED_uuid });
+    test("When start_calibration is called, Then playback state mutates to calibrating and starts status_pinging in Flask, then playback state mutates to calibrated", async () => {
+      let baseurl = "http://localhost:4567";
```

Comment:
Looks like `baseurl` and `api is common in multiple places, can we have them defined on the most exterior describe beforeEach function.